src/test/java/edu/ucla/library/bucketeer/verticles/ClearCacheVerticleTest.java

```diff
@@ -0,0 +1,153 @@
+
+/**
+ * Testing the mechanism used to clear Cantaloupe's cache.
+ */
+@RunWith(VertxUnitRunner.class)
+public class ClearCacheVerticleTest {
+
+    static {
+        // For running this test independently, we need to configure the logging config source
+        System.setProperty("logback.configurationFile", "src/test/resources/logback-test.xml");
+    }
+
+    /**
+     * Logger for the <code>ClearCacheVerticleTest</code>.
+     */
+    private static final Logger LOGGER = LoggerFactory.getLogger(ClearCacheVerticleTest.class, Constants.MESSAGES);
+
+    /**
+    * A test context from which references to Vert.x can be retrieved.
+    */
+    @Rule
+    public RunTestOnContext myRunTestOnContextRule = new RunTestOnContext();
+
+    /**
+     * Cantaloupe username for testing Cantaloupe cache clearing.
+     */
+    private String myUsername;
+
+    /**
+     * Cantaloupe password for testing Cantaloupe cache clearing.
+     */
+    private String myPassword;
+
+    /**
+     * Verticle ID for undeploying ClearCacheVerticle
+     */
+    private String myVertID;
+
+    /**
+     * Sets up the tests.
+     *
+     * @param aContext A testing environment
+     */
+    @Before
+    public final void setup(final TestContext aContext) {
+        final ConfigRetriever configRetriever = ConfigRetriever.create(Vertx.vertx());
+        final Async asyncTask = aContext.async();
+
+        configRetriever.getConfig(configuration -> {
+            if (configuration.failed()) {
+                aContext.fail(configuration.cause());
+            } else {
+                final JsonObject config = configuration.result();
+                final DeploymentOptions options = new DeploymentOptions().setConfig(config);
+
+                myRunTestOnContextRule.vertx()
+                    .deployVerticle(ClearCacheVerticle.class.getName(), options, deployment -> {
+                        if (deployment.succeeded()) {
+                            myVertID = deployment.result();
+                            TestUtils.complete(asyncTask);
+                        } else {
+                            aContext.fail(deployment.cause());
+                        }
+
+                    });
+            }
+        });
+
+
+    }
+
+
+    /**
+     * Tear down the testing environment.
+     *
+     * @param aContext A test context
+     */
+    @After
+    public void tearDown(final TestContext aContext) {
+
+        if (myVertID != null) {
+            final Async async = aContext.async();
+            myRunTestOnContextRule.vertx().undeploy(myVertID, undeployment -> {
+                if (undeployment.failed()) {
+                    aContext.fail(undeployment.cause());
+                } else {
+                    async.complete();
+                }
+            });
+        } else {
+            LOGGER.info(MessageCodes.BUCKETEER_605);
+        }
+
+
+
+       //in tear down get verticle id and shut down the verticle by referencing the ID
```

Comment:
Clean up extra new lines. I wonder if this is the best place for this last comment too? This seems like something that could go into the method's Javadocs?